detection-rules/spam_single_recipient_duplicated_in_cc.yml

```diff
@@ -0,0 +1,95 @@
+name: "Spam: Single recipient duplicated in cc"
+description: "Detects spam emails where the 'To' and 'CC' fields match, using indicators such as short body length with spam keywords, unsolicited content, dmarc failures, fake threads, and suspicious links."
+type: "rule"
+severity: "medium"
+source: |
+   type.inbound
+   
+  // one recipient and 1 cc
+  and length(recipients.to) == 1
+  and length(recipients.cc) == 1
+  
+  // unsolicited
+  and not profile.by_sender().solicited
+  
+  // recipients to equals cc
+  and any(recipients.to, any(recipients.cc, .email.email == ..email.email))
+  
+  // body is short with spam keywords
+  and (
```

Comment:
am a little worried about FPs since it only needs to match 1 of these conditions. we should probably test rules it?

Reply:
unsolicted + 1 recipient and 1 cc and they match is almost good enough to ship on it's own.

Reply:
Rule has 150+ matches in with an AS of 90+